pkg/virt-controller/network/multus_annotations_test.go

```diff
@@ -62,7 +63,7 @@ var _ = Describe("Multus annotations", func() {
 
 		It("when added an element, is no longer empty", func() {
 			podIfaceName := "net1"
-			multusAnnotationPool.add(newMultusAnnotationData(vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, network, podIfaceName))
+			multusAnnotationPool.add(multus.NewAnnotationData(vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, network, podIfaceName))
```

Comment:
Can we move this tests to pkg/network/multus as well?

Reply:
This test is moved in the commit moving the `multusNetworkAnnotationPool`.